src/client/openfinGlobal.ts

```diff
@@ -0,0 +1,7 @@
+import { fin as MockFin } from 'openfin-adapter/src/mock';
```

Comment:
should not need this .. let's sync up later, I think we may need to swap notes between this and the OpenFin upgrade branch 😄

Reply:
we'll come back to this when I merge in the OF upgrade